src/openapi/streaming/subscription.js

```diff
@@ -263,6 +290,32 @@ function onUnsubscribeError(referenceId, response) {
 }
 
 /**
+ * Called after modify patch is successful
+ * @param referenceId
+ * @param response
+ */
+function onModifyPatchSuccess(referenceId, response) {
+    if (referenceId !== this.referenceId) {
+        log.debug(LOG_AREA, 'Received an error response for patching a subscription');
+        return;
+    }
+    onReadyToPerformNextAction.call(this);

```

Comment:
you do not need to do this because you do not change state into something pending, so you are not releasing anything